code/modules/clothing/spacesuits/void/void_vr.dm

```diff
@@ -1,25 +1,60 @@
+//
+// Because of our custom change in update_icons, we cannot rely upon the normal
+// method of switching sprites when refitting (which is to have the referitter
+// set the value of icon_override).  Therefore we use the sprite sheets method
+// instead.
+//
+
 /obj/item/clothing/head/helmet/space/void
 	sprite_sheets = list(
 		"Tajara" 				= 'icons/mob/species/tajaran/helmet.dmi',
 		"Skrell" 				= 'icons/mob/species/skrell/helmet.dmi',
 		"Unathi" 				= 'icons/mob/species/unathi/helmet.dmi',
+		"Teshari"				= 'icons/mob/species/seromi/head.dmi',
 		"Nevrean" 				= 'icons/mob/species/nevrean/helmet_vr.dmi',
 		"Akula" 				= 'icons/mob/species/akula/helmet_vr.dmi',
 		"Sergal"				= 'icons/mob/species/sergal/helmet_vr.dmi',
 		"Flatland Zorren" 		= 'icons/mob/species/fennec/helmet_vr.dmi',
 		"Highlander Zorren" 	= 'icons/mob/species/fox/helmet_vr.dmi'
 		)
-
-
+	sprite_sheets_obj = list(
```

Comment:
Interestingly, I had figured that would be an issue, and already added it to mine. 

I'll fix the merge commit myself since this has more added on.